build/js/flow.js

```diff
@@ -7427,7 +7427,7 @@
         exportNotebook = function () {
             var remoteName;
             if (remoteName = _remoteName()) {
-                return window.open('/3/NodePersistentStorage.bin/notebook/' + remoteName, '_blank');
+                return window.open(window.Flow.ContextPath + '3/NodePersistentStorage.bin/notebook/' + remoteName, '_blank');
```

Comment:
1) do not modify files in /build these are compiled files only commited when new version of Flow is released

2) Flow has undergone a modernization, please rebase your change and modify correct .coffee file

3) use _.ContextPath instead of window.Flow.ContextPath (was refactored)